doc/developers/contributing.rst

```diff
@@ -140,11 +140,7 @@ feedback:
   your **Python, scikit-learn, numpy, and scipy versions**. This information
   can be found by running the following code snippet::
 
-     import platform; print(platform.platform())
-     import sys; print("Python", sys.version)
-     import numpy; print("NumPy", numpy.__version__)
-     import scipy; print("SciPy", scipy.__version__)
-     import sklearn; print("Scikit-Learn", sklearn.__version__)
+     import sklearn; sklearn.show_versions()
```

Comment:
In the stable documentation I think we would need to have separate paragraphs for scikit-learn >=0.20 and <0.20 here. Otherwise users with older versions will not report relevant system information at all.